<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>618</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ROUNDNET</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page 1</string>
       </attribute>
       <widget class="QLineEdit" name="JsonKey">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>124</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>potVex</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>63</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Val. desiree</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>30</y>
          <width>3</width>
          <height>556</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>0</y>
          <width>218</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Messages Json de l'Arduino:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>121</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Valeur Ki</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>310</y>
          <width>241</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>État 0 : Initialisation du robot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>État 1 : Oscillation aller</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>État 2 : Passe par dessus l'obstacle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>État 3 : Arrêt de l'oscillation aller</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>État 4 : Lache le sapin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>État 5 : Reviens au début</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>État 6 : Arrêt oscillation retour</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Kd">
        <property name="geometry">
         <rect>
          <x>677</x>
          <y>155</y>
          <width>99</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_DesVal">
        <property name="geometry">
         <rect>
          <x>677</x>
          <y>65</y>
          <width>99</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>263</x>
          <y>250</y>
          <width>0</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxPort">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>7</y>
          <width>124</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>121</y>
          <width>94</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Duree (ms)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>187</y>
          <width>94</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Donnees brutes:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Kp">
        <property name="geometry">
         <rect>
          <x>677</x>
          <y>94</y>
          <width>99</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Params">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>230</y>
          <width>221</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoie Parametres</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pulseButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>225</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Commande de pulse</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>281</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>92</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Valeur Kp</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>36</y>
          <width>225</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enregistrement des donnees sous:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_pathCSV">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>63</y>
          <width>225</width>
          <height>22</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>185</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Seuil</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>7</y>
          <width>94</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>92</y>
          <width>94</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Tension [-1,1]</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>7</y>
          <width>256</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Execice PID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="Stop">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>270</y>
          <width>150</width>
          <height>28</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="DurationBox">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>121</y>
          <width>124</width>
          <height>22</height>
         </rect>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>25</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ki">
        <property name="geometry">
         <rect>
          <x>677</x>
          <y>123</y>
          <width>99</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>567</x>
          <y>150</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valeur Kd</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Thresh">
        <property name="geometry">
         <rect>
          <x>677</x>
          <y>187</y>
          <width>99</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="StartButton">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>270</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="PWMBox">
        <property name="geometry">
         <rect>
          <x>111</x>
          <y>92</y>
          <width>124</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
       <widget class="QChartView" name="graph">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>531</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Page 2</string>
       </attribute>
       <widget class="QGraphicsView" name="Graphique">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>641</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Distance obstacle</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Distance dépôt</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>341</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Robot en temps Réel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="distanceObstacle">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="hauteurObstacle">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="envoiDistance">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoie distance</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="distanceDepot">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Hauteur obstacle</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
